aws/network/bastion/main.tf

```diff
@@ -22,13 +22,29 @@ variable "owner" {
   default = ""
 }
 
+variable "open_sesame" {
```

Comment:
I suggested the rename. On seeing it, I think you're right @kbarek, that is should not really have much reference to open sesame. It should just state that it needs other means of opening the security groups if this is set.